client/src/components/Survey.tsx

```diff
@@ -13,14 +13,20 @@ export interface SurveyProps {
 const SURVEY = 'SurveyComponent';
 
 export const Survey: React.FunctionComponent<SurveyProps> = (props: SurveyProps) => {
+  const surveyType = props.postCall.survey?.type;
+  let postcallSurveyUrl = '';
+  if (surveyType === 'msforms') {
+    const options: MSFormsSurveyOptions = props.postCall.survey?.options as MSFormsSurveyOptions;
+    postcallSurveyUrl = options.surveyUrl;
+  } else if (surveyType === 'custom') {
+    const options: CustomSurveyOptions = props.postCall.survey?.options as CustomSurveyOptions;
+    postcallSurveyUrl = options.surveyUrl;
+  } else {
+    postcallSurveyUrl = '';
+  }
   return (
     <Stack styles={surveyStyle}>
-      <iframe
-        title={SURVEY}
-        style={surveyIframeStyle}
-        src={props.postCall.survey?.options.surveyUrl}
-        scrolling="yes"
-      ></iframe>
+      <iframe title={SURVEY} style={surveyIframeStyle} src={postcallSurveyUrl} scrolling="yes"></iframe>
```

Comment:
I think we first have to build the UI for one question poll before we make it the default option in the config, if I am not mistaken. Otherwise, when the client gets the config and sees that post call is set to one question poll, it doesn't have yet the UI to handle that.

Reply:
Yep, I agree, so we will not make one question poll the default option for now.